ops/op2/generator_state.rs

```diff
@@ -4,8 +4,6 @@ use proc_macro2::Ident;
 pub struct GeneratorState {
   /// Identifiers for each of the arguments of the original function
   pub args: Vec<Ident>,
-  /// The new identifier for the original function's contents.
-  pub call: Ident,
```

Comment:
Not exactly sure why, but now clippy complains that this field is never used. @mmastrac can you take a look and confirm?

Reply:
Done in https://github.com/denoland/deno_core/pull/680